Common/Product/SharedProject/CommonUtils.cs

```diff
@@ -320,39 +227,31 @@ public static string GetRelativeDirectoryPath(string fromDirectory, string toDir
         /// </summary>
         public static string GetRelativeFilePath(string fromDirectory, string toFile)
         {
-            var fromUri = MakeUri(fromDirectory, true, UriKind.Absolute, "fromDirectory");
-            var toUri = MakeUri(toFile, false, UriKind.Absolute, "toFile");
+            var splitDirectory = Path.GetFullPath(TrimEndSeparator(fromDirectory)).Split(DirectorySeparators);
+            var splitFile = Path.GetFullPath(toFile).Split(DirectorySeparators);
```

Comment:
Consider adding a comment about why it's okay to include the file name in the path part comparison.  (I agree that it is, it just wasn't obvious why.)